externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/FlinkOperation.scala

```diff
@@ -98,25 +105,43 @@ abstract class FlinkOperation(session: Session) extends AbstractOperation(sessio
     assertState(OperationState.FINISHED)
     setHasResultSet(true)
     order match {
-      case FETCH_NEXT => resultSet.getData.fetchNext()
+      case FETCH_NEXT =>
```

Comment:
Is it changed to doing nothing for `FETCH_NEXT` ? 
Move to the last condition if it's necessary. Or it is misleading a fallback situation
(or use  `case FETCH_NEXT | FETCH_PRIOR` for falling back)

Reply:
> Is it changed to doing nothing for FETCH_NEXT ?
 
Yes. Now `resultSet.getData.fetchNext()` is invoked lazily for Flink, when the fetched batch is fully consumed.

Reply:
Thanks for the explaination.